pyvo/samp.py

```diff
@@ -10,9 +10,9 @@
 import tempfile
 
 try:
-    from astropy.vo.samp import SAMPIntegratedClient
-except ImportError:
     from astropy.samp import SAMPIntegratedClient
+except ImportError:
+    from astropy.vo.samp import SAMPIntegratedClient
```

Comment:
Is this another one of those it was in one place in one version of astropy and somewhere different in another version?  If so, is this the time to use the minversion thing to make it a bit more explicit?

Reply:
that's possible sure, I simply kept the original logic but fixed the usage of it.